compiler/circle-eval-diff/README.md

```diff
@@ -0,0 +1,41 @@
+# circle-eval-diff
+
+_circle-eval-diff_ compares inference results of two circle models.
+
```

Comment:
maybe add some typical example of this tool?
I assume this will be used before-after quantization of a cirle model which first and second model data types are different, thus input data files have different data types according.